hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -512,25 +509,18 @@ public Option<HoodieInstant> getFirstNonSavepointCommit() {
   }
 
   @Override
-  public Option<HoodieInstant> getLastClusterCommit() {
-    return  Option.fromJavaOptional(getCommitsTimeline().filter(s -> s.getAction().equalsIgnoreCase(HoodieTimeline.REPLACE_COMMIT_ACTION))
+  public Option<HoodieInstant> getLastCompleteOrPendingClusteringInstant() {
```

Comment:
Kind of think the name `getLastClusteringInstant` should be fine, by default it includes all the states of clustering.

Reply:
Makes sense.  Fixed.